CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+- 2023-10-25 - v0.4.4
+    - using release v0.4.6 of lightarti-rest
+    - updated to Android SDK 34
+    - pinned Android NDK 25 (cf. https://github.com/bbqsrc/cargo-ndk/issues/94)
```

Comment:
Did I misunderstand something? Did you pin it in the project itself, so it's not necessary anymore in the github-workflow?

Reply:
yes